### ISSUE TITLE
Update 3.0.x to 3.0.2

### ISSUE THREAD

**Issue by ralscha** [os72/protoc-jar#11]
Hi

There is a version 3.0.2 of the protobuf compiler
http://repo1.maven.org/maven2/com/google/protobuf/protoc/3.0.2/

Maybe it's worth updating the binaries to this version, although there is not much information in the changelog: https://github.com/google/protobuf/releases/tag/v3.0.2

Interestingly google released version 3.1.0 of the protobuf library, but no new compiler artifacts in this version: https://github.com/google/protobuf/releases/tag/v3.1.0

**Comment by os72** [comment]
Ack. 3.1.0 binaries are there too
http://repo1.maven.org/maven2/com/google/protobuf/protoc/3.1.0/

**Comment by os72** [comment]
Released protoc-jar-3.1.0

**Comment by avakhrenev** [comment]
`protoc-jar-3.1.0` fails in our CI environment:
```
/tmp/protocjar1452708085854790798/bin/protoc.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protocjar1452708085854790798/bin/protoc.exe)
```
Problem seems to be known: https://github.com/google/protobuf/issues/781#issuecomment-252026657

Could you please update binaries from `protoc-3.1.0-build2`?

**Comment by os72** [comment]
Ack. Yes

**Comment by os72** [comment]
commit 198ca6ba2271159549852d27ea8be526d1602362

**Comment by os72** [comment]
Released 3.1.0.1
